ik:
  chain_start: "base_link"
  chain_end: "tool"
  timeout: 0.005
  epsilon: 1e-5

fk:
  joint_origins:
    - [0.0, 0.0, 0.1595]
    - [0.000154, -0.0525, 0.167346]
    - [0.0, -0.0105, 0.28]
    - [0.0455, 0.063, 0.01]
    - [0.2545, -0.0235, 0.0]
    - [0.056, 0.0235, 0.0]
    - [0.09025, 0.0, 0.0]
  joint_axes:
    - [0.0, 0.0, 1.0]
    - [0.0, 1.0, 0.0]
    - [0.0, 1.0, 0.0]
    - [1.0, 0.0, 0.0]
    - [0.0, 1.0, 0.0]
    - [1.0, 0.0, 0.0]
    - [0.0, 0.0, 0.0]
